test/SlowTests/Server/Documents/ETL/SQL/SqlEtlTests.cs

```diff
@@ -1179,6 +1180,18 @@ await session.StoreAsync(new Order
             }
         }
 
+        [RequiresMsSqlRetryFact(delayBetweenRetriesMs: 1000)]
+        public void CanTestConnection()
+        {
+            using (var store = GetDocumentStore())
+            {
+                using (SqlAwareTestBase.WithSqlDatabase(MigrationProvider.MsSQL, out var connectionString, out string schemaName, dataSet: null, includeData: false))
+                {
+                    SqlDatabaseWriter.TestConnection("Microsoft.Data.SqlClient", connectionString);
```

Comment:
Don't we need to have dedicated test for testing other providers as well?

Reply:
Thought about that, but added just one - the code is almost 1:1, just going through different cases in DbProviderFactories. 
However, I can add it to ensure all cases work. Will do

Reply:
added also oracle, npgsql, and mysql